shell/browser/api/electron_api_system_preferences_win.cc

```diff
@@ -117,6 +166,24 @@ std::string SystemPreferences::GetColor(gin_helper::ErrorThrower thrower,
   return ToRGBHex(color_utils::GetSysSkColor(id));
 }
 
+std::string SystemPreferences::GetMediaAccessStatus(
+    const std::string& media_type,
+    gin_helper::Arguments* args) {
+  if (media_type == "camera") {
+    return ConvertDeviceAccessStatus(
+        GetDeviceAccessStatus(DeviceClass::DeviceClass_VideoCapture));
+  } else if (media_type == "microphone") {
+    return ConvertDeviceAccessStatus(
+        GetDeviceAccessStatus(DeviceClass::DeviceClass_AudioCapture));
+  } else if (media_type == "screen") {
+    return ConvertDeviceAccessStatus(
+        DeviceAccessStatus::DeviceAccessStatus_Allowed);
```

Comment:
We should note in the documentation that `screen` access is always allowed on Windows, i'd say.